src/main/resources/META-INF/rust-core.xml

```diff
@@ -521,10 +521,15 @@
                          enabledByDefault="true" level="ERROR"
                          implementationClass="org.rust.ide.inspections.RsTraitImplementationInspection"/>
 
-        <localInspection language="Rust" groupName="Rust"
-                         displayName="Match Check"
+        <localInspection language="Rust" groupPath="Rust" groupName="Lints"
+                         displayName="Useless match arm"
+                         enabledByDefault="true" level="WARNING"
+                         implementationClass="org.rust.ide.inspections.checkMatch.RsUselessMatchArmInspection"/>
+
+        <localInspection language="Rust" groupPath="Rust" groupName="Lints"
```

Comment:
Oh, I don't think we should move `RsNonExhaustiveMatchInspection` to `Lints` group. Only `RsUselessMatchArmInspection`. Yeah, they are very similar from implementation side but not from user point of view: non-exhaustive `match` is a compilation error and it even has own number in error index. At the same time, an unreachable pattern is warning by default and can be suppressed by user.
So let's revert this change

Reply:
Sure, I did it accidentally, thanks for your attention!